pages/login_page.rb

```diff
@@ -1,31 +1,40 @@
 require_relative "./page.rb"
-
 # would deal with anny assertions, logging, etc
 
 
 
 class LoginPage < Page
     # UI MAPPING
-    USERNAME_FIELD = "CSS SELECTOR";
-    PASSWORD_FIELD = "DSFADSF";
-
-    def initialize url, driver
+    USERNAME_FIELD = { :name => "email"}
+    PASSWORD_FIELD = { :name => "password"}
+    LOGIN_BUTTON =   { :name => "button"}
+    LOGIN_PATH = "/login"
+        
+    # Builds the LoginPage
+    def initialize url:, driver:, wait:
         super
         puts "navigating to login page if not here"
+        @driver.get @url + LOGIN_PATH
     end
 
+    # enters the password in the password field
     def enter_password password
-        puts "types password into field"      
+        puts "types password into field" 
+        password_field = @wait.until { @driver.find_element(PASSWORD_FIELD) }
```

Comment:
remove the `wait.until` here. Sending keys 99% of the time will have fields already visible and writable once the page loads

Reply:
Moved "Find element and send keys" to page, without the wait
 this will be addressed in next push
Leaving it up to selenium to throw an error for no element found